README.md

```diff
@@ -22,6 +20,13 @@ likely to be useful:
  - Adding a new third-party code or service dependency.
  - Changing review workflows or policies.
 
+Cases where the RFC process need *not* be used:
+
+ - Introducing a new top-level directory for a new specification.
+ - Minor changes in behavior in where all call sites are known and accounted
+   for.
+ - Behavior-preserving refactoring with a low risk of regressions.
+
 As a rule of thumb, please use the RFC process when it would be useful and err
```

Comment:
I don't really understand this rule of thumb. It doesn't seem to be a simple but imprecise test one can apply to figure out the correct decision (i.e. a "rule of thumb"), but a general request to use the process when appropriate. 

The correct rule of thumb is probably closer to "please use the RFC process for changes likely to have an impact on multiple users of wpt".

Reply:
This section leads with "Use this process for substantial changes which might impact other stakeholders or users of web-platform-tests", that's roughly the same as "likely to have an impact on multiple users of wpt".

You're right that this isn't a rule of thumb, I've pushed an additional commit that I think is an improvement, but please review: https://github.com/web-platform-tests/rfcs/pull/2/commits/56fc69c820798cb419292f43aab9da61565314ec